radix-engine/src/blueprints/consensus_manager/validator.rs

```diff
@@ -465,6 +481,7 @@ impl ValidatorBlueprint {
             LockFlags::read_only(),
         )?;
         let validator: ValidatorSubstate = api.field_read_typed(handle)?;
+        api.field_close(handle)?;
```

Comment:
not sure if this was missed or skipped on purpose?

Reply:
some tests fail after adding this, so I guess this must have been on purpose :man_shrugging: , reverted

Reply:
The field needs to remain open in order to have access to the vaults within. The system will close the field automatically. Otherwise, if you'd like you can close it at the end of the method but not required.